### PR TITLE
Added create new comment

### PR DESCRIPTION
**Pull request by dwb264**

### CODE REVIEW

**Comment by steveleee** [comment]
How can I test this out to see if it works?

File: dispatch_app/views.py [hunk 1]
```diff
@@ -8,6 +9,26 @@
 from dispatch_app.serializers import *
 
 # Create your views here.
+@csrf_exempt
+@permission_classes([IsAuthenticated])
+@api_view(['POST'])
+def comment(request, tweet_id):
+    try:
+        tweet = Tweet.objects.get(tweet_id=tweet_id)
```
Comment (steveleee) [inline dispatch_app/views.py]: I think this line throws the exception, do we still need the next line?

File: dispatch_app/views.py [hunk 2]
```diff
@@ -8,6 +9,26 @@
...
+        serializer = CommentSerializer(data=data, context={'request': request})
+
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        else:
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    except Tweet.DoesNotExist:
+        return Response({"error": "Tweet does not exist"}, status=status.HTTP_204_NO_CONTENT)
```
Comment (steveleee) [inline dispatch_app/views.py]: Could you make this error message more descriptive? It should help users of the API debug their requests. Following that, I think `HTTP_400` would be more appropriate here.

File: dispatch_app/views.py [hunk 3]
```diff
@@ -8,6 +9,26 @@
 from dispatch_app.serializers import *
 
 # Create your views here.
+@csrf_exempt
+@permission_classes([IsAuthenticated])
+@api_view(['POST'])
+def comment(request, tweet_id):
```
Comment (steveleee) [inline dispatch_app/views.py]: Docs for this function please

**Comment by dwb264** [comment]
I made the following changes:

- Added a test case for creating a comment with an existing and non-existing tweet_id
- Removed the unnecessary line in comment function
- Added description of comment function
- Made comment function error more specific and changed error code to 400

**Comment by steveleee** [comment]
This is simply awesome. 👍

**Pull request merged**